init.lua

```diff
@@ -498,6 +499,7 @@ function _M.switch(dir, mod_key1, release_key, mod_key2, key_switch)
 						-- raise chosen client on top of all
 						c = _M.altTabTable[_M.altTabIndex].client
 						c:raise()
+            c:jump_to()
```

Comment:
Here too, the indentation is off compared to surrounding lines.

Reply:
Fixed the indentation.